src/components/Modals/Onboarding/OnboardingBasicSingleDropdown.vue

```diff
@@ -89,8 +89,10 @@ export default defineComponent({
       // scroll the bottom of the graduation year dropdown to scrollBottomToIndex
       if (!contentShown && this.scrollBottomToIndex > 0) {
         this.$nextTick(() => {
-          const el = (this.$refs.scrollRef as Element[])[this.scrollBottomToIndex];
-          el.scrollIntoView({ behavior: 'auto', block: 'nearest' });
+          (this.$refs[`scroll-ref-${this.scrollBottomToIndex}`] as Element).scrollIntoView({
```

Comment:
Can you make this same change to the code for the onboarding graduation year dropdown? I forgot I made that autoscroll last-minute as well.

Reply:
Yes +1 to what @willespencer said! Currently, when clicking the dropdown for graduation year in Onboarding, the dropdown doesn't autoscroll to 2021 and I get a type error appear in the console. Note that this doesn't show up when I click on any other dropdown in Onboarding. The graduation year does seem to save though.

https://user-images.githubusercontent.com/54298311/114028676-be228c80-9846-11eb-9314-50da6b64e55d.mov